### PR TITLE
Fix times, links, and git URL

### PR DESCRIPTION
**Pull request by PaulWessel**
This branch should finalize the times for each sub-section and match the times in the [Google Docs paper](https://docs.google.com/document/d/1a_ZS3Z_4Eht2CfJQiwB3e19ppUuWCQBcV2inGclTBhQ/edit) (well, off by 1 hour since Oxford vs UTC).

I also added Federico's GitHub URL - @Esteban82, please update if you want another URL here.

@meghanrjones, could you update the [form](https://forms.gle/FnEeTK9CLgYHBMzD9) we ask them to fill out to say 2022?

All instructors: Please check the schedule again as some of the durations have changed a bit again.

### CODE REVIEW

**Comment by Esteban82** [comment]
This can be merge, right?

**Comment by PaulWessel** [comment]
I want to wait for Max to update the link to her survey form first.

File: README.md [hunk 1]
```diff
@@ -70,42 +71,39 @@ Participants will receive the zoom invitation at least 24 hours before the start
...
-| Wrap up      | 10 min |  21:50-22:00  |
+| Welcome   | 10 min | 14:30-14:40  |
+| [Seismology](seismology) | 80 min |  14:40-16:00  |
+| *Break*      | 60 min | 16:00-17:00  |
+| [Geodesy](geodesy)  | 70 min |  17:00-18:10  |
+| *Break*      | 10 min |  18:10-18:20 |
+| [Animations](animation)      | 65 min | 18:20-19:25  |
+| Wrap up      | 5 min |  19:25-19:30  |
 
 ### Day 3:
```
[the remark this answers is not in the document]
Comment (PaulWessel) [reply README.md:96]: They have different names and content.  What do you mean - similar?

**Comment by PaulWessel** [comment]
> 

Hi @meghanrjones, when you get to this please approve after updating the form URL.

**Review by meghanrjones — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Esteban82) [reply README.md:96]: Sorry. I was looking at tthe version in the main branch.

**Pull request merged**